Doc/library/codecs.rst

```diff
@@ -697,8 +697,8 @@ compatible with the Python codec registry.
 
    .. method:: writelines(list)
 
-      Writes the concatenated list of strings to the stream (possibly by reusing
-      the :meth:`write` method). The standard bytes-to-bytes codecs
+      Concatenates the string iterable and writes it to the stream. Infinite or
```

Comment:
I don't like the way the rest of the docs was reworded. It's not clear to a reader what "the string iterable" is. Let's just leave the first sentence unchanged.

Reply:
Is `Writes the concatenated iterable of strings to the stream` better or `Writes the concatenated string iterable to the stream`?

Reply:
I prefer the first one; it reads more clearly.